.github/workflows/ci-cd.yml

```
@@ -58,12 +58,20 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
 
       - name: Initialize CodeQL
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 61: could not find expected ':'

(syntax-check)